src/models/localization/fil-tl.js

```diff
@@ -0,0 +1,107 @@
+const facts = [
+  "Noong 1987, naunahan ng mga pusa ang aso bilang pinaka-sikat na alagang hayop sa America",
```

Comment:
If you ever want to change these facts to make them fit the language's and local's culture more, its very appropriate and encouraged. 

As an example this first fact could be changed to anything non-america 

> According to a survey in the philippines on pet ownership conducted by Rakuten Insight in January 2022, dogs were the most popular pets, with 81 percent of respondents saying they own a Dog over a Cat.

Reply:
feel free to come back later and make additional contributions if you have some interesting facts as well :clap: